src/main/resources/schema/ans/0.10.0/story_elements/header.json

```diff
@@ -21,7 +21,9 @@
     "alignment": {
       "$ref": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.0/traits/trait_alignment.json"
     },
-
+    "format": {
```

Comment:
I think header should not have this element

Reply:
I was originally discussed to have it but I am fine with removing that as well